addons/sale_coupon/tests/test_program_numbers.py

```diff
@@ -1546,3 +1546,46 @@ def test_fixed_amount_change_promo_amount(self):
 
         self.assertEqual(len(order.order_line), 1, 'Promotion line should not be present')
         self.assertEqual(order.amount_total, 10, '10$ - 0$(discount) = 10$(total) ')
+
+    def test_fixed_tax_not_affected(self):
+        self.env['coupon.program'].create({
+            'name': '50% discount',
+            'program_type': 'promotion_program',
+            'promo_code_usage': 'no_code_needed',
+            'reward_type': 'discount',
+            'discount_type': 'percentage',
+            'discount_percentage': 50,
+            'active': True,
```

Comment:
Nitpicking: I guess this is useless since it's default value :slightly_smiling_face: